### DISCUSSION TITLE
V5 Stl question

### DISCUSSION THREAD

**Discussion by ducky92** [sanni/cartreader#440] [General Chat]
I am in the process of drawing up a new V5 stand and does anyone know if these "half-round" sections in the file near the corners are needed for anything? I do not want to remove them if they are needed.
![image](https://user-images.githubusercontent.com/10150745/181352993-81947355-cb68-4f17-ae45-87924fb3e117.png)

My current design.
![Selection_465](https://user-images.githubusercontent.com/10150745/181353146-b12cd469-3672-415e-abb1-a47101e8b565.png)

**Comment by Ancyker** [comment]
They look like they are intended for those rubberized non-scuff/slip feet such as you'd find on AV equipment.

Edit: Yeah, the rubber feet are actually on the parts list so that's what they are for.

**Comment by Ancyker** [comment]
Also, I know you didn't ask but that looks really good. I was actually concerned about the current design not being rigid enough and since I don't have a 3D printer to try it out I decided to just get it printed in a different material (nylon). Yours looks a lot sturdier. Do you plan to add walls and make it more of an enclosure? 

I have only one criticism: Why's it say V9 though? lol

**Reply by ducky92** [reply]
This is the 9th version I have printed out with minor changes between them while I am trying to get all the measurements correct. At some point it was hard to tell and I added it. 

Yes I actually plan on pushing a step file to the repo once I get this all ironed out. I really hate that that STL has so many errors. 

I have printed the original and this new one in PETG and they both seem fine. My current one has more supports so its a little stiffer. 

I think if I widen the front corners I will be golden. You wouldn't happen to have one of those feet you could tell me the size of it do you? I don't think I ordered them.

**Reply by Ancyker** [reply]
The ones the wiki links to are 8mm.

**Comment by ducky92** [comment]
V10 is on the printer; I should have something to hold in about 3 hours. I made the front sides a little wider. V9 above had a 6mm frame, it now has a 9mm one (the round pieces appear to be 8mm half circles)
![image](https://user-images.githubusercontent.com/10150745/181379345-2c9485e8-93fc-4bcd-9f97-50ce27688cdb.png)

The left side is designed to work with this [N64 connector](https://www.aliexpress.com/item/2251832657756075.html?gatewayAdapt=4itemAdapt)
here is a different angle 
![image](https://user-images.githubusercontent.com/10150745/181379804-d8802b37-81d6-470b-a1db-1434eea73bd8.png)

**Reply by Ancyker** [reply]
Oh, that's really cool. Any chance to put it on the other side though? They sent me the displays that have the [SD card slot facing front](https://i.imgur.com/6iJTUzB.jpg).

**Reply by ducky92** [reply]
The problem with the right side is that it interferes with the display connectors unless it sticks out really far. Let me play around with it some more and I can see what I can come up with.

**Reply by Ancyker** [reply]
Ah, that makes sense. I'm still waiting on everything to come so I haven't actually assembled it yet.

**Reply by Icesythe7** [reply]
If taking the time to redesign the frame then make the holder for the n64 port the actual shape of the n64 port so it doesn't need glued in place, or at least attempt some type of proper mounting system for it. (just my 2 cents) Looks great otherwise.

**Reply by ducky92** [reply]
> If taking the time to redesign the frame then make the holder for the n64 port the actual shape of the n64 port so it doesn't need glued in place, or at least attempt some type of proper mounting system for it. (just my 2 cents) Looks great otherwise.

Did you not look at the image you commented on? With this current design the N64 connector snaps into place with very little wiggling and is a PITA to try to get back out. I am trying to iron out the connector being on the opposite side when life allows.

**Comment by ducky92** [comment]
These changes have beep added to the main repo https://github.com/sanni/cartreader/tree/master/hardware/main_pcb_hw5

**Comment by ducky92** [comment]
@Ancyker I have something that may work printing now. 
![image](https://user-images.githubusercontent.com/10150745/181961898-c59b67be-ccb5-4d52-ab24-ef7785fb2e30.png)

**Reply by ducky92** [reply]
@Ancyker you may need to trim the legs on the knob with some flush cutters but this should work (there is an open pull request for Sanni's repo; hopefully its available shortly) 
https://github.com/ducky92/cartreader/blob/Frame-STL-Revision/hardware/main_pcb_hw5/Main%20PCB%20Stand%20-%20Right%20N64%20Adapter.stl

**Reply by Icesythe7** [reply]
That looks nice thus far! I thought about designing one with a "top" more of a fully enclosed box style which would allow for the N64 port to be mounted to the side of the box aswell but haven't done anything yet because I like looking at the pcbs aswell haha, thought about aswell on the v5 just moving the gb adapter to the left/right a bit and adding a 90degree N64 port right there just soldered straight to the board

**Reply by Ancyker** [reply]
@ducky92 thanks so much. I had already ordered the original design before you posted this (the original thread) and it hasn't even come yet, lol. I do plan to get my own 3D printer later in the year so I might wait for that to try out this design. I'll see how stable the old design feels first.

**Reply by Ancyker** [reply]
@Icesythe7 I was thinking having one with sides but no top and then using laser cut acrylic for the top would be cool.

**Reply by Icesythe7** [reply]
That would be super cool
